packages/nx/src/plugins/js/lock-file/pnpm-parser.spec.ts

```diff
@@ -112,47 +112,99 @@ describe('pnpm LockFile utility', () => {
     let graph: ProjectGraph;
     let lockFile: string;
 
-    beforeEach(() => {
-      lockFile = require(joinPathFragments(
-        __dirname,
-        '__fixtures__/nextjs/pnpm-lock.yaml'
-      )).default;
-      const builder = new ProjectGraphBuilder();
-      parsePnpmLockfile(lockFile, builder);
-      graph = builder.getUpdatedProjectGraph();
+    describe('v5.4', () => {
+      beforeEach(() => {
+        lockFile = require(joinPathFragments(
+          __dirname,
+          '__fixtures__/nextjs/pnpm-lock.yaml'
+        )).default;
+        const builder = new ProjectGraphBuilder();
+        parsePnpmLockfile(lockFile, builder);
+        graph = builder.getUpdatedProjectGraph();
+      });
+
+      it('should parse root lock file', async () => {
+        expect(Object.keys(graph.externalNodes).length).toEqual(1280); ///1143
```

Comment:
What does this comment mean?

Reply:
It's the number of hoisted packages, which was important information in the debugging of the earlier versions.

I'll remove it since it doesn't bring any value right now and it's cryptic to everyone else.